code/modules/assembly/voice.dm

```diff
@@ -28,7 +28,10 @@
 		return
 
 	if(listening)
-		recorded = msg
+		if(findtext(msg, "</span>"))
+			recorded = strip_html_properly("[msg]")
```

Comment:
```suggestion
			recorded = strip_html_properly(msg)
```
Should always already be a string.

Reply:
Also, any reason not to always do the strip_html?

Reply:
actually no reason, dont know why i didnt think about that